rs/sns/init/src/lib.rs

```diff
@@ -3471,61 +3444,25 @@ mod test {
         }
         // Test that if the constraints are violated, the validation, indeed, fails.
         {
-            let fdvp = FractionalDVP {
-                swap_distribution: Some(SwapDistribution {
-                    initial_swap_amount_e8s: MAX_DIRECT_ICP_CONTRIBUTION_TO_SWAP,
-                    // Not used in this test.
-                    total_e8s: 0,
-                }),
-                // Not used in this test.
-                developer_distribution: None,
-                treasury_distribution: None,
-                airdrop_distribution: None,
-            };
             let sns_init_payload = SnsInitPayload {
-                max_direct_participation_icp_e8s: Some(MAX_DIRECT_ICP_CONTRIBUTION_TO_SWAP),
-                min_direct_participation_icp_e8s: Some(1),
                 // This value being so low ensures there would be too many SNS neurons.
```

Comment:
Having a low `min_participant_icp_e8s` does not guarantee a certain number of participants.
Only a low `max_participant_icp_e8s` can do that, right?

Reply:
There are indirect dependencies between all these parameters, but you're correct that `min_participant_icp_e8s` is no longer directly influencing the possible number of participants.

Reply:
Ah, you are pointing to the comment. Indeed, it should be adjusted. Thanks

Reply:
Done